.github/workflows/build_packages.yaml

```diff
@@ -151,7 +151,23 @@ jobs:
       with:
         ref: ${{ github.event.inputs.ref }}
         fetch-depth: 0
-    - name: build emqx packages
+    - name: build tgz
+      env:
+        PROFILE: ${{ matrix.profile }}
+        ARCH: ${{ matrix.arch }}
+        OS: ${{ matrix.os }}
+        IS_ELIXIR: ${{ matrix.with_elixir }}
+        BUILDER: "ghcr.io/emqx/emqx-builder/${{ matrix.builder }}:${{ matrix.elixir }}-${{ matrix.otp }}-${{ matrix.os }}"
+        BUILDER_SYSTEM: force_docker
+      run: |
+        ./scripts/buildx.sh \
+          --profile $PROFILE \
+          --arch $ARCH \
+          --builder $BUILDER \
+          --elixir $IS_ELIXIR \
+          --pkgtype tgz
+    - name: build pkg
+      if: matrix.with_elixir == 'no'
```

Comment:
This will exclude the single build that contains Elixir from having a `.deb` package.  Is this intentional?

Reply:
Yes, I replicated previous behavior https://github.com/emqx/emqx/blob/release-55/.github/workflows/build_packages.yaml#L177

```
if [ "${IS_ELIXIR:-}" == 'yes' ]; then
  make "${PROFILE}-elixir-tgz"
else
  make "${PROFILE}-tgz"
  make "${PROFILE}-pkg"
fi
```

Frankly, I'm not sure what was the motivation to not build system packages for emqx with elixir.